v2/cmd/samples/kafka/sender-receiver/main.go

```diff
@@ -4,18 +4,25 @@ import (
 	"context"
 	"fmt"
 	"log"
+	"sync/atomic"
 
 	"github.com/Shopify/sarama"
 
 	cloudevents "github.com/cloudevents/sdk-go/v2"
 	"github.com/cloudevents/sdk-go/v2/protocol/kafka_sarama"
 )
 
+const (
+	count = 10
+)
+
 func main() {
 	saramaConfig := sarama.NewConfig()
+	saramaConfig.Producer.Return.Successes = true
```

Comment:
Mh now that i see that, I think it just makes sense to force this value directly in production code, here:

* https://github.com/cloudevents/sdk-go/blob/master/v2/protocol/kafka_sarama/sender.go#L19
* https://github.com/cloudevents/sdk-go/blob/master/v2/protocol/kafka_sarama/protocol.go#L42